docs/dev.md

```diff
@@ -139,6 +139,24 @@ If on Linux, it is recommended that you copy the entire `configDropins/overrides
 COPY --chown=1001:0  target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides /config/configDropins/overrides
 ```
 
+##### Specify Liberty configuration with Maven properties
+
+If you are using `liberty.var.{var}` or `liberty.defaultVar.{var}`, then the server configuration file named `liberty-plugin-variable-config.xml` is generated in the configDropins/overrides folder or configDropins/defaults folder of the target server respectively.
+To ensure your application runs correctly inside a container, you need to copy this generated liberty-plugin-variable-config.xml file into your container image. 
+
+You can do this by adding one of the following lines to your Dockerfile or Containerfile, depending on which type of variable you used.
+To copy the file from the overrides folder:
+
+```dockerfile
+COPY --chown=1001:0  target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/liberty-plugin-variable-config.xml /config/configDropins/overrides/liberty-plugin-variable-config.xml
+```
+To copy the file from the defaults folder:
+
+```dockerfile
+COPY --chown=1001:0  target/liberty/wlp/usr/servers/defaultServer/configDropins/defaults/liberty-plugin-variable-config.xml /config/configDropins/defaults/liberty-plugin-variable-config.xml
```

Comment:
All the other copy command examples I have seen do not specify the file name for the target location but instead just specify the directory. Shouldn't we use the same pattern here?

Reply:
Also, for the example before this one it has special instructions for Linux. There must have been a reason for that. Does that apply here as well? Since both sections deal with configDropins, I thought it would make more sense to combine the sections and talk about the need to copy over generated files.